test/e2e/Makefile

```diff
@@ -37,7 +37,7 @@ export GOBIN := $(abspath $(TOOLS_BIN_DIR))
 
 ginkgo: $(GINKGO)
 $(GINKGO): # Build ginkgo from tools folder.
-	go install github.com/onsi/ginkgo/v2/ginkgo@v2.11.0
+	go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
it this intended? (I don't know whether it makes sense, I'm literally asking)

Reply:
strange. This is from a PR which was merged by Anurag. The new (green) line is the one which is correct. I did a `git pull origin/main` to get the latest changes. Don't ask me why github shows it in the diff.

Reply:
Maybe because you merged the main branch in here or something? Can you clean up the PR and squash everything, then we can see again. But it shouldn't show like this if all git stuff worked correctly

Reply:
@janiskemper I rebase on main again, and now the change at `-mod=mod` is gone. I think there is no need to wait for the e2e tests to finish. Ready for merge?

Reply:
I would like them to finish, yes, in case that this is messing up the provisioning process, they are very valid!

Reply:
problem is again that this was not in draft, so you triggered with e2e tests multiple times.. they don't get canceled if you push again on the same PR

Reply:
e2e tests were successful.